### PR TITLE
camellia: Initial official support

### PR DESCRIPTION
**Pull request by rexix01**

### CODE REVIEW

File: camellia.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "response": [
+    {
+        "maintainer": "Sabrina",
+        "oem": "Xiaomi",
```
Comment (gwolf2u) [inline camellia.json]: go with Redmi

File: camellia.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        "device": "Redmi Note 10 5G/Note 10T/Note 11 SE (china), Poco M3 Pro",
```
Comment (gwolf2u) [inline camellia.json:6]: try limiting to max 25 chars
previously was "RN10 5G/10T/11SE/M3 Pro"

File: camellia.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        "filename": "crDroidAndroid-14.0-20240303-camellia-v10.2.zip",
+        "download": "",
```
Comment (gwolf2u) [inline camellia.json:8]: missing download url
use generated one

File: camellia.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        "buildtype": "Monthly",
+        "forum": "",
+        "gapps": "https://nikgapps.com/crdroid-official",
+        "firmware": "https://xiaomifirmwareupdater.com/firmware/camellian/",
+        "modem": "",
+        "bootloader": "",
+        "recovery": "",
+        "paypal": "",
+        "telegram": "https://t.me/pecamellia",
+        "dt": "https://github.com/crdroidandroid/android_device_xiaomi_xamellia",
```
Comment (gwolf2u) [inline camellia.json:23]: bad url

File: camellia.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        "common-dt": "",
+        "kernel": "https://github.com/crdroidandroid/android_kernel_xiaomi_camellia"
```
Comment (gwolf2u) [inline camellia.json:25]: missing 14.0 branch

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: camellia.json (hunk 1 above)
Comment (rexix01) [reply camellia.json]: Okay

**Pull request closed without merging**